specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json

```diff
@@ -12716,9 +12771,9 @@
           {
             "in": "query",
             "name": "source",
-            "description": "Source name of scene data, default value Sentinel_2_L2A (Sentinel 2 L2A).",
-            "type": "string",
-            "default": "Sentinel_2_L2A"
+            "description": "Source name of scene data, Available Values: Sentinel_2_L2A, Sentinel_2_L1C.",
+            "required": true,
+            "type": "string"
```

Comment:
prefer extensible enum https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#enums--sdks-client-libraries

Reply:
Facing same issue as with path params. Unable to use enums in query parameters. We are using swashbuckle to generate swagger and when using enums in query, type info is not getting populated.